### PR TITLE
Release Karaf Runtime 4.2.7

### PR DESCRIPTION
**Pull request by fpapon**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: src/main/webapp/download.html [hunk 1]
```diff
@@ -70,19 +70,19 @@ <h2 class="pb-3 mb-2"><i class="fas fa-download"></i> Download</h2>
           <div class="card flex-md-row mb-4 box-shadow h-md-250">
             <div class="card-body d-flex flex-column align-items-start">
               <strong class="d-inline-block mb-2 text-success"><i class="fas fa-circle"></i> Latest release</strong>
-              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.6</span></h3>
-              <div class="mb-1 text-muted">June 10 2019</div>
+              <h3 class="mb-0 text-dark">Karaf Runtime <span class="text-muted">4.2.7</span></h3>
+              <div class="mb-1 text-muted">September 30 2019</div>
```
Comment (jbonofre) [inline src/main/webapp/download.html]: Release date is actually on Sep 29th.

**Review by jbonofre — CHANGES_REQUESTED** [review]
Can you please update the release date to Sep 29th ? Else I will do it.

File: src/main/webapp/download.html (hunk 1 above)
Comment (fpapon) [reply src/main/webapp/download.html]: Ok I do it now ;)

Comment (fpapon) [reply src/main/webapp/download.html]: @jbonofre done!

**Review by jbonofre — APPROVED** [review]
Thanks ! I will merge as soon as the artifacts will be sync on mirrors.

**Pull request merged**